MdePkg/Test/Mock/Include/GoogleTest/Protocol/MockFirmwareVolumeBlock.h

```diff
@@ -0,0 +1,103 @@
+/** @file MockFirmwareVolumeBlock.h
+  This file declares a mock of Firmware Volume Block Protocol
+
+  Copyright (c) Microsoft Corporation.
+  Your use of this software is governed by the terms of the Microsoft agreement under which you obtained the software.

```

Comment:
This should be replaced with the SPDX license identifier in all files.

```suggestion
SPDX-License-Identifier: BSD-2-Clause-Patent
```